pages/menu/edit-customer/[id].js

```diff
@@ -0,0 +1,57 @@
+import Header from "../../../components/Header/Header";
+import { Form, FormButton } from "../../../pagestyles/styles";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+
+export default function EditCustomer() {
+  const router = useRouter();
+  const { id } = router.query;
+  const { mutate } = useSWR("/api");
+
+  async function handleEditSubmit(e, id) {
```

Comment:
```suggestion
  async function handleEditSubmit(event, id) {
```
For better readability, you should avoid shorthand shortcuts.